src/workerd/api/filesystem.h

```diff
@@ -374,18 +374,11 @@ class FileSystemFileHandle final: public FileSystemHandle {
 };
 
 class FileSystemDirectoryHandle final: public FileSystemHandle {
- public:
-  struct EntryType {
-    jsg::USVString key;
-    jsg::Ref<FileSystemHandle> value;
-    JSG_STRUCT(key, value);
-    EntryType(jsg::USVString key, jsg::Ref<FileSystemHandle> value)
-        : key(kj::mv(key)),
-          value(kj::mv(value)) {}
-  };
-
  private:
-  using EntryIteratorType = EntryType;
+  // The actual entry type is an array of two elements, the first being the key,
+  // the second being the value.
+  using EntryType = kj::OneOf<jsg::JsRef<jsg::JsString>, jsg::Ref<FileSystemHandle>>;
```

Comment:
Why replace that explicit EntryType struct with a comment explaining what that struct looks like?

Reply:
Not sure what you mean. The struct was wrong. The entry type here is an array whose first element is a string and whose second element is the handle.

Reply:
Ah. I see.